### DISCUSSION TITLE
Download a JSON file

### DISCUSSION THREAD

**Discussion by fnagel** [inertiajs/inertia#2050] [Help (React)] [answered]
Can somebody help me on how to download a file? In my case a JSON file? 

My current code (which will only do a redirect): `<Link target="_blank" download href={route('project.export', project.id, true)}>Export</Link>`
Using an onclick event works but looks hackish to me: `onClick={() => {window.open(route('project.export', project.id as number, true))}}`

Any advise what is the proper solution here?

**Comment by jefhar** [comment, marked as answer]
`<a target="_blank" download href={route('project.export', project.id, true)}>Export</a>`

**Reply by fnagel** [reply]
Thanks for your reply! This works well too!

Can you explain why its working with `<a>` but not with `<Link>`?

**Reply by jefhar** [reply]
`<Link>` is for an Inertia link. Downloading a file, or linking to another website, or telephone number, or email link doesn't go through Inertia, it's just an `<a>`, because it's normal HTML.
